helm/examples/certmanager/README.md

```diff
@@ -11,14 +11,6 @@ The following steps can be used to set up cert-manager in a kubernetes cluster.
 
 --------------------------------------------------------------------------------------------------------
 
-## Install cert-manager custom resource defintions:
```

Comment:
Adding --set installCRDs=true to the helm install line below means we don't have to separately apply the CRDs ourselves.